services/madoc-ts/src/frontend/shared/caputre-models/VerboseSelector.tsx

```diff
@@ -21,7 +22,26 @@ export const VerboseSelector: React.FC<VerboseSelectorProps> = ({ selector, read
 
   useEffect(() => componentWillUnmount, [componentWillUnmount]);
 
+  const updateSelector = useCallback(
+    state => {
+      if (selector) {
+        selector.state = state;
+        updateSelectorValue({ selectorId: selector.id, state });
```

Comment:
I think this may break in some cases, I think this would fit better in the `updateSelectorValue` method itself here:
https://github.com/digirati-co-uk/capture-models/blob/master/packages/editor/src/stores/revisions/revisions-store.tsx#L280-L334

So it can go through the existing tests.

Reply:
The problem with this suggestion is that the selector that drives the component (i.e. `selector` here, which goes into `useSelectorStatus`) isn't the same object as the one being available in `updateSelectorValue` (which is looked up from the ID). When this change goes into capture models instead of here, the fix no longer works because the selector that drives the component has the wrong state on it